custom_components/onesmartcontrol/alarm_control_panel.py

```diff
@@ -84,13 +85,13 @@ def __init__(
 
     @property
     def supported_features(self) -> AlarmControlPanelEntityFeature:
-        return AlarmControlPanelEntityFeature.ARM_AWAY | AlarmControlPanelEntityFeature.ARM_NIGHT
+        return AlarmControlPanelEntityFeature.ARM_HOME | AlarmControlPanelEntityFeature.ARM_AWAY | AlarmControlPanelEntityFeature.ARM_NIGHT
```

Comment:
I don't think we should mark the 'home' mode of One Smart Control as 'Armed'. IMO if the HomeKit integration does not support 'disarmed', which conforms to Home Assistant's specification of the entity, this should be fixed in the HomeKit integration and not here.